src/libServer/IsolatedServer.cpp

```diff
@@ -619,6 +619,8 @@ Json::Value IsolatedServer::CreateTransaction(const Json::Value& _json) {
 
 Json::Value IsolatedServer::CreateTransactionEth(EthFields const& fields,
                                                  bytes const& pubKey) {
+  Json::Value ret;
```

Comment:
nitpick: any reason we moved this variable to the beginning of the method?

Reply:
It now returns outside of the try... catch and so also has to be declared in that scope. It needs to return outside the scope it was originally defined in because there is a mutex that is locked in that scope which is also used by PostTxBlock (so would deadlock).